addons/account/models/partner.py

```diff
@@ -427,3 +427,13 @@ def _commercial_fields(self):
         return super(ResPartner, self)._commercial_fields() + \
             ['debit_limit', 'property_account_payable_id', 'property_account_receivable_id', 'property_account_position_id',
              'property_payment_term_id', 'property_supplier_payment_term_id', 'last_time_entries_checked']
+
+    @api.multi
+    def action_open_partner_history(self):
+        '''
+        This function returns an action that display invoices/refunds made for the given partners.
+        '''
+        self.ensure_one()
+        action = self.env.ref('account.action_invoice_refund_out_tree').read()[0]
+        action['domain'] = [('partner_id', 'in', self.ids)]
```

Comment:
That's idiotic. Why are you taking a list of ids after having ensured the action was only called on a single object? Is the action supposed to be called on a single partner or multiple?